src/mame/drivers/atarittl.cpp

```diff
@@ -192,6 +197,23 @@ class stuntcyc_state : public driver_device
 	double m_last_fraction;
 };
 
+class gtrak10_state : public driver_device
+{
+public:
+	gtrak10_state(const machine_config &mconfig, device_type type, const char *tag)
+		: driver_device(mconfig, type, tag)
+	{
+	}
+
+	void gtrak10(machine_config &config);
+
+protected:
+        // driver_device overrides
+        virtual void machine_start() override { };
+        virtual void machine_reset() override { };
+        virtual void video_start() override  { };
```

Comment:
There's no point overriding these if you're not doing anything.  Also you have superfluous semicolons.